src/main/java/core/basesyntax/service/implementation/ReadFromFileServiceImplementation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.implementation;

```

Comment:
the same for package, do not use full word "Implementation", fix everywhere
```suggestion
package core.basesyntax.service.impl;
```